docs/spark-qualification-tool.md

```diff
@@ -6,12 +6,25 @@ nav_order: 8
 
 # Qualification Tool
 
-The Qualification tool analyzes Spark events generated from CPU based Spark applications to determine 
-if the RAPIDS Accelerator for Apache Spark might be a good fit for GPU acceleration.
+The Qualification tool analyzes Spark events generated from CPU based Spark applications to help quantify
+the expected acceleration of migrating a Spark application to GPU.
+
+The tool first analyzes the operators executed from the CPU log and determines which ones would be qualified to run on
+GPU.  
+The tool then uses estimates from historical queries and benchmarks to estimate a speed-up at an individual operator
+level to calculate how much a specific operator would accelerate on GPU for the specific application.  
+To calculate the _"Estimated GPU App Duration"_, it will add up the accelerated operator durations along with durations
+that could not run on CPU because they are unsupported operators or not SQL operations.
```

Comment:
It calculates an _"Estimated GPU App Duration"_ by adding up ...  

could not run on GPU... not SQL/Dataframe